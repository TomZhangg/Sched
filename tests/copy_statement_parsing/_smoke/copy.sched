Copy Schedule x Of Schedule y;


Copy Schedule Item x Of Schedule Item y;


Copy Schedule Collection x Of Schedule Collection y;

