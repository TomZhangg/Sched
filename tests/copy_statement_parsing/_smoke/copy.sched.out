<copy-statement>
  <copy-sched-of-sched-statement>
    <id>: Id(x)
    <id>: Id(y)
<copy-statement>
  <copy-item-of-item-statement>
    <id>: Id(x)
    <id>: Id(y)
<copy-statement>
  <copy-coll-of-coll-statement>
    <id>: Id(x)
    <id>: Id(y)
